src/app/components/shell/header/header.scss

```diff
@@ -44,182 +73,587 @@
+                    right: -2rem;
+                    top: 50%;
+                    width: 0;
+
+                    @include desktop {
+                        height: 0.7rem;
                         right: 1rem;
+                        margin-top: -0.7rem;
                     }
                 }
 
-                @at-root .transparent#{&} {
-                    color: color(neutral-lightest);
+                &::after {
+                    margin-top: -1px;
+                    transform: rotate3d(0, 0, 1, 45deg);
 
-                    &:active,
-                    &:focus,
-                    &:hover {
-                        &::after {
-                            box-shadow: 0 0 0.1rem rgba(color(neutral-dark), 0.6);
-                        }
+                    @include desktop {
+                        margin-top: -0.7rem;
+                        right: 0.6rem;
                     }
                 }
             }
-        }
+
+            &:first-child > a {
+                @include desktop {
+                    margin-left: -2rem;
+                }
+            }
+
+            > a {
+                white-space: nowrap;
+
+                @include desktop {
+                    padding-left: 2rem;
+                    padding-right: 2rem;
+                }
+          }
+      } //end .nav-menu-item
     }
 
-    .logo-wrapper {
-        flex: 0 0 auto;
-        line-height: 0;
+    //Navigation with Rice Logo row
+    .meta-nav {
+        order: 2;
+
+        @include desktop {
+            order: 1;
+            background: color(neutral-lighter);
+            height: 5rem;
+
+            @at-root .sticky#{&} {
+                position: fixed;
+                top: -7rem;
+                left: 0;
+                right: 0;
+                z-index: 1;
+                height: 7rem;
+                transform: translate3d(-5rem, 0, 0);
+                background: transparent;
+          }
+      }
+
+        .container {
+            @include desktop {
+                display: flex;
+                flex-direction: row nowrap;
+                justify-content: flex-end;
+                height: 100%;
+                align-items: center;
+            }
+      }
+
+        .nav-menu {
+            padding: 0 0 0 8rem;
+            margin: 0;
+            list-style: none;
+            flex: 1 100%;
+
+            @media (max-width: #{breakpoints(tablet)})  {
+                @at-root .active.open#{&} {
+                    transform: translate3d(-1000px, 0, 0);
+                    transition: transform 0.5s;
+                }
+            }
+
+            @include desktop {
+                flex: 0 0 auto;
+                display: flex;
+                padding: 0;
 
-        span {
-            vertical-align: middle;
-        }
+                @at-root .sticky.visible.active#{&} {
+                    transform: translate3d(0, 7rem, 0);
+                    transition: transform 0.5s 0.6s;
```

Comment:
Some of these delayed transforms make the nav bar disappear for a period of time on scroll and act a little clunky.  Is this the intended behavior?